lib/espree.js

```diff
@@ -232,7 +252,7 @@ export default () => Parser => {
 
         /**
          * Overwrites the default raise method to throw Esprima-style errors.
-         * @param {int} pos The position of the error.
+         * @param {number} pos The position of the error.
```

Comment:
Just wondering, this isn't related to `acorn.Position` is it?

Reply:
No, this was presumably done to simplify the fact that TS (and JS) doesn't distinguish regular integers and the number type. I think it would be ideal to preserve this though so as to clarify the intent, e.g., for autocomplete.

Reply:
But then passing a number wouldn't satisfy TS's typechecker, no?
I'd rather have it be integer too but I think we can mention that in the parameter description since TS/JS don't have native integers.

Reply:
`int` would need to have its own `typedef` set to `number` (but if it's a problem because TypeScript will export the type, don't worry about it).